content/understanding-urbit/urbitos1-osn.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "Urbit OS1 –> OSN"
```

Comment:
OS¹ -> OSⁿ? Hehe.

Reply:
Yeah, I'd personally prefer this styling tbh

Reply:
Up to you